src/main/java/org/elasql/server/Elasql.java

```diff
@@ -59,7 +60,11 @@ public class Elasql extends VanillaDb {
 	private static Logger logger = Logger.getLogger(VanillaDb.class.getName()); 
  
 	public static final long START_TX_NUMBER = 0; 
-	public static final long START_TIME_MS = System.currentTimeMillis(); 
+	public static final long START_TIME_MS = System.currentTimeMillis();
+	public static boolean testMode = false;

```

Comment:
I suggest to use a enum with `{NOT_INITIALIZED, NORMAL, TEST}` to represent the state of ElaSQL.